Query.php

```diff
@@ -551,7 +551,7 @@ public function query($query)
     public function batch($scrollWindow = '1m', $db = null)
     {
         return Yii::createObject([
-            'class' => BatchQueryResult::className(),
+            'class' => BatchQueryResult::class,
```

Comment:
You've fixed not all constants. Please check ["Files changed" tab](https://github.com/yiisoft/yii2-elasticsearch/pull/255/files).

Reply:
Yep. Saw those. I think I got all of them now.